src/main/java/seedu/duke/commands/DisplayCalendarCommand.java

```diff
@@ -0,0 +1,40 @@
+package seedu.duke.commands;
+
+import seedu.duke.exceptions.InvalidArgumentsException;
+import seedu.duke.storage.Storage;
+import seedu.duke.ui.Ui;
+import seedu.duke.parser.Parser;
+import seedu.duke.schedule.Schedule;
+import java.time.YearMonth;
+
+public class DisplayCalendarCommand extends Command {
+
+    private int year;
+    private int month;
+    private YearMonth inputYearMonth;
+    private String[] yearMonthArguments;
+
+    public DisplayCalendarCommand(String input) {
+        try {
+            try {
+                this.yearMonthArguments = Parser.parseCalendarCommand(input);
+            } catch (IndexOutOfBoundsException | NumberFormatException e) {
+                Ui.printInvalidYearMonthMessage();
+            }
+            this.year = Integer.parseInt(yearMonthArguments[1]);
+            this.month = Integer.parseInt(yearMonthArguments[0]);
+            if (year < 2021 || year > 2025 || month < 1 || month > 12) {
```

Comment:
`year > 2025`
Any reason why 2025 is used as an upper limit?

Reply:
Thought I should introduce a limit considering that the user could give an input with year as 20000 or 1870 and it would still display.
Would you suggest I just remove this check?

Reply:
I see, was wondering that 2025 applied for the target audience (nus students graduating by 2025) - maybe replace magic literals `2021`, `2025` as  constants in `constants` package instead?

Reply:
It's just a check to prevent nonsensical inputs (for now).
Accepted and implemented suggestion.